cves/CVE-2013-0306.yml

```diff
@@ -77,13 +81,24 @@ vccs:
 - commit: 7adffaeaf6dfa22db0b6b2a29632b9150c7ac732
   note: This VCC was discovered automatically via archeogit.
 - commit: aba95dcc0b5370ffac3d3b701c3ca7782ee999c1
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
```

Comment:
Can remove the automatically discovered line for all the commits because you actually discovered the commits.

Reply:
I reviewed the commits that were automatically discovered and removed those that did not benefit my research.